examples/metadata_processing/plot_add_dict_entries.py

```diff
@@ -1,49 +1,75 @@
 """
-=====================================================
-Add dictionary items in the standard DICOM dictionary
-=====================================================
-
-This examples illustrates how to add dictionary items to the 'standard' DICOM
-dictionary dynamically.
-
-.. warning::
-
-   Note that this is not a recommended solution in general but it is useful as
-   a demonstration or for in-house programs only, or to get around elements
-   missing from pydicom's DICOM dictionaries.
+=============================================================
+Add private dictionary items to the standard DICOM dictionary
```

Comment:
This is a bit confusing, since its not the standard dictionary

Reply:
Thanks - missed this while adapting the title.

Reply:
Changed the wording